CMakeLists.txt

```diff
@@ -45,7 +45,23 @@ if(CMAKE_BUILD_TYPE STREQUAL Coverage)
     find_package(Coverage REQUIRED)
 endif()
 
-option(BUILD_DOCUMENTATION "Build documentation" ON)
+###############################################################################
+# Documentation
+###############################################################################
+
+option(BUILD_DOCUMENTATION          "Build documentation" ON)
```

Comment:
I don't think the BUILD_DOCUMENTATION option should be on by default. I should be set by the user when running cmake. This is why the CI build is failing, because the docker used by Travis doesn't have doxygen installed.

Reply:
Done.